float64 drivetrain_fwd_back
float64 drivetrain_left_right
float64 drivetrain_swerve_percent_fwd_vel
float64 drivetrain_swerve_direction
float64 drivetrain_swerve_percent_angular_rot
bool drivetrain_quickturn
bool drivetrain_brake
int32 gauge_value
float64 elevator_vertical
bool claw_open